articles/azure-resource-manager/management/manage-resource-groups-portal.md

```diff
@@ -46,7 +46,7 @@ The resource group stores metadata about the resources. Therefore, when you spec
 
     ![browse resource groups](./media/manage-resource-groups-portal/manage-resource-groups-list-groups.png)
 
-3. To customize the information displayed for the resource groups, select **Edit columns**. The following screenshot shows the addition columns you could add to the display:
+3. To customize the information displayed for the resource groups, select **Edit columns**. The following screenshot shows the additional columns you could add to the display:
```

Comment:
```suggestion
3. To customize the information displayed for the resource groups, select **Edit columns**.
```
There is no screenshot following this step, so either @mumian should add one or the sentence that refers to it should be removed.